pkg/services/sqlstore/user.go

```diff
@@ -345,12 +345,18 @@ func GetUserOrgList(query *m.GetUserOrgListQuery) error {
 	return err
 }
 
-func GetSignedInUser(query *m.GetSignedInUserQuery) error {
+func (ss *SqlStore) GetSignedInUser(query *m.GetSignedInUserQuery) error {
 	orgId := "u.org_id"
 	if query.OrgId > 0 {
 		orgId = strconv.FormatInt(query.OrgId, 10)
 	}
 
+	cacheKey := fmt.Sprintf("signed-in-user-%d-%s", query.UserId, query.OrgId)
```

Comment:
@torkelo do we need to use both org id and user id in cache key, isn't it enough with user id?

Reply:
The cached obj contains orgId and org role

Reply:
The cached obj contains orgId and org role

Reply:
Ah yes now I understand! Good point.